packages/runtime/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# `react`
```

Comment:
I know you copied this package, but make sure to use the right package name here and every where else needed.